tentacoli.js

```diff
@@ -291,6 +294,21 @@ Tentacoli.prototype.request = function (data, callback) {
   return this
 }
 
+Tentacoli.prototype.fire = function (data, callback) {
+  var that = this
+  var req = new Request(null, callback)
+
+  try {
+    wrapStreams(that, data, req, true)
+  } catch (err) {
+    return this
```

Comment:
Otherwise we could accept a callback and use it just for user errors.

```js
sender.fire('message', err => { if (err) console.log(err) })
```
In this way we demand to the user the error control, and if the user does not give us a callback we'll use a `noop`.